README.rst

```diff
@@ -0,0 +1,55 @@
+WebAuthn Authenticator Plugin
+=============================
+.. image:: https://travis-ci.org/curityio/webauthn-authenticator.svg?branch=master
+    :target: https://travis-ci.org/curityio/webauthn-authenticator
+    
+This project provides an open source WebAuthn Authenticator plug-in for the Curity Identity Server. This allows an administrator to add functionality to Curity which will then enable end users to login using their physical authenticator devices.
+
+Building the Plugin
+~~~~~~~~~~~~~~~~~~~
+
+You can build the plugin by issue the command ``gradlew build``. This will produce a JAR file in the target directory, which can be installed.
+
+Installing the Plugin
+~~~~~~~~~~~~~~~~~~~~~
+
+To install the plugin, copy the compiled JAR (and all of its dependencies) into the ``${IDSVR_HOME}/usr/share/plugins/${pluginGroup}`` on each node, including the admin node. For more information about installing plugins, refer to the `curity.io/plugins`_.
+Some of the dependencies required by the plugin are already provided by the Curity Identity Server. The additional ones that are not provided and need to be copied together with the plugin are:
+
+* org.apache.commons
+* com.google.guava
+* com.fasterxml.jackson.core
+* com.fasterxml.jackson.dataformat
+* org.jose4j (This dependency is going to be provided in version 5.1.0 and above of the Curity Identity Server)
+
+You can find the dependencies jar files in ``build/dist/`` after executing the task ``gradlew copyRuntimeLibs``. Note that some of the above dependencies bring in a few extra ones.
```

Comment:
I have to do this:
```
mkdir ~/id/dist/usr/share/plugins/webauthn
cp build/dist/* $IDSVR_HOME/usr/share/plugins/webauthn
cp build/libs/webauthn-authenticator-1.0.0.jar $IDSVR_HOME/usr/share/plugins/webauthn
```
Why not put the resulting JAR and its dependencies in the same place?

Reply:
In the build/libs directory of other plugins we only have the jar of the plugin, however I do agree there isn't any compelling reason against having everything under buid/libs. I will change this.

Reply:
We should look into changing those. One dir is better. Thanks for updating it